core/src/main/java/com/novoda/accessibility/ClosedCaptionManagerCompat.java

```diff
@@ -0,0 +1,61 @@
+package com.novoda.accessibility;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.os.Build;
+import android.view.accessibility.CaptioningManager;
+
+public class ClosedCaptionManagerCompat {
+
+    public static CaptionManager newInstance(Context context) {
```

Comment:
if ClosedCaptionManagerCompat doesn't implement CaptionManager, then this should not be called `newInstance(Context)`. It should be something like `createCaptionManager(Context)` or `newCaptionManager(Context)`